nifi-commons/nifi-record/src/main/java/org/apache/nifi/serialization/record/ResultSetRecordSet.java

```diff
@@ -26,12 +27,12 @@
 import java.math.BigDecimal;
 import java.math.BigInteger;
 import java.sql.Array;
+import java.sql.Date;
```

Comment:
Changing `java.util.Date` to `java.sql.Date` changes the value of `DATE_CLASS_NAME`, does that have any other implications that should be considered?

Reply:
It was intentional because it seemed to be a bug. `ResultSet.getMetaData().getColumnClassName()` would never return `java.util.Date` but the SQL date/time types.
However, now I see that this code section is related to QueryRecord and it may use `java.util.Date`. I'll revert it back.
Thanks for the heads up that made me double check it in more detail.

Reply:
Thanks for responding regarding the column class name handling.  I will take another look once you push the next set of updates.